src/common.browser/WebsocketConnection.ts

```diff
@@ -79,11 +78,11 @@ export class WebsocketConnection implements IConnection {
             headers);
     }
 
-    public dispose = (): void => {
+    public async dispose(): Promise<void> {
```

Comment:
>public async dispose(): Promise<void> { [](start = 4, length = 39)

I think we've inherited some class designs from the C++ code here in a way that doesn't make sense for JS. We don't really have any native resources to "dispose" of here and I think a cleaner design would be to rename this to close() which makes more sense as async function.

Reply:
While I don't disagree, next time.

---
In reply to: [457669739](https://github.com/microsoft/cognitive-services-speech-sdk-js/pull/198#discussion_r457669739) [](ancestors = 457669739)